docs/mocksupport.rst

```diff
@@ -0,0 +1,131 @@
+.. _`Mock support`:
+
+Mock support
+============
+
+The pywbem PyPI package provides unit testing support for its users via its
+``pywbem_mock`` Python package. This package allows users of pywbem to
+define FakedWBEMConnections and CIM objects (classes, qualifier declarations,
+and instances) that will be processed by these connections. In effect this
+creates a fake WBEM Server with limited capabilities.
+
+The faked HMC environment is set up by creating an instance of the
+:class:`pywebm_mock.FakeWBEMConnection` class instead of the
```

Comment:
Change "FakeWBEMConnection" to "FakedWBEMConnection".

Occurs multiple times in the .rst file and in the .py files.

Reply:
DONE